SqlDataCatalog-Collibra/convert-to-collibra.ps1

```diff
@@ -0,0 +1,203 @@
+            relations    = [pscustomobject]@{
+                "00000000-0000-0000-0000-000000007005:TARGET" = @($databaseIdentifier)
+            }
+        }
+
+        $_.group | Group-Object tableName | ForEach-Object {
+            # table
+
+            $tableName = $_.Name
+            $tableIdentifier = [pscustomobject]@{
+                name   = "$instanceName>$databaseName>$schemaName>$tableName";
+                domain = $physicalModelIdentifier
+            };
+            $export += [pscustomobject]@{
+                resourceType = "Asset";
+                identifier   = $tableIdentifier;
+                displayName  = $tableName;
+                type         = [pscustomobject]@{
+                    name = "Table"
+                };
+                status       = $acceptedStatus;
+                relations    = [pscustomobject]@{
+                    "00000000-0000-0000-0000-000000007043:SOURCE" = @($schemaIdentifier)
+                }
+            }
+
+            $_.group | ForEach-Object {
+                # column
+
+                $columnName = $_.columnName
+                $tagNames = @()
+                Foreach ($tag in $_.tags) {
+                    $tagNames += ($tag.name.Replace(" ", "_"))
+                }
+                $columnIdentifier = [pscustomobject]@{
+                    name   = "$instanceName>$databaseName>$schemaName>$tableName>$columnName";
+                    domain = $physicalModelIdentifier
+                };
+
+                # Attributes
+                $attributes = [pscustomobject]@{}
+
+                $securityClassificationTag = $tagNames.Where({ $securityClassificationMappings.ContainsKey($_) }, 'First')
+                if($securityClassificationTag) {
+                    $attributes | Add-Member -MemberType NoteProperty -Name "Security Classification" -Value @(
+                        [pscustomobject]@{
+                            value = $securityClassificationMappings[$securityClassificationTag][0]
+                        }
+                    )
+                }
+
+                $isPii = ($personallyIdentifiableInformationTags.Where({ $tagNames.Contains($_) }, 'First').Count -gt 0)
+                $attributes | Add-Member -MemberType NoteProperty -Name "Personally Identifiable Information" -Value  @(
+                    [pscustomobject]@{
+                        value = $isPii
+                    }
+                )
+
+                $export += [pscustomobject]@{
+                    resourceType = "Asset";
+                    identifier   = $columnIdentifier;
+                    displayName  = $columnName;
+                    type         = [pscustomobject]@{
+                        name = "Column"
+                    };
+                    status       = $acceptedStatus;
+                    relations    = [pscustomobject]@{
+                        "00000000-0000-0000-0000-000000007042:TARGET" = @($tableIdentifier)
```

Comment:
Is this a known constant value in Collibra, or do we need to make some sort of comment to change the value here?